src/EventGridEmulator/Dockerfile

```diff
@@ -1,9 +1,11 @@
 ﻿FROM mcr.microsoft.com/dotnet/aspnet:7.0-alpine AS base
 WORKDIR /app
-EXPOSE 8080
+EXPOSE 5000
 ENV DOTNET_CLI_TELEMETRY_OPTOUT=true
 ENV DOTNET_ENVIRONMENT=Production
-ENV ASPNETCORE_URLS=http://+:8080
+ENV ASPNETCORE_URLS=http://+:5000
+ENV DOCKER_HOST=tcp://localhost:2375
+ENV DOCKER_DRIVER=overlay
```

Comment:
Are those two necessary?

Reply:
Maybe something I don't understand but it wasn't working without that.

Reply:
I didn't really tought too much about the code. I focused on the pipeline. Should we create a task to revisit the code?

Reply:
Do you know how to make it work without it?